src/main.ts

```diff
@@ -25,7 +23,7 @@ function getJestAdapterOptions(): JestTestAdapterOptions {
   };
 }
 
-export async function activate(context: vscode.ExtensionContext) {
+export function activate(context: vscode.ExtensionContext): void {
```

Comment:
I see a few of these removals of `async` from a few functions.  That has effectively changed the signature to return `void` instead of `Promise<void>`.  This might have consequences, especially here where this method is called by vscode.

Reply:
I'll test it. ESlint flagged it as 'async without await'